src/charactersheet/viewmodels/dm/encounter_sections/point_of_interest_section/index.html

```diff
@@ -141,6 +141,33 @@
               </span>
             </div>
           </div>
+          <div class="form-group ui-front">
+            <label class="col-sm-2 control-label">
+              Required DC
```

Comment:
Initially, it felt a little weird to put this in POI, but upon lookingat our options, there really isn't any better section.